### DISCUSSION TITLE
Criei um Scrum Poker real time 😁

### DISCUSSION THREAD

**Discussion by allanzi** [frontendbr/forum#2067] [Mostre seu trabalho]
Opa, fala pessoal.

Queria mostrar pra vcs uma ferramenta que fiz, devido a pandemia e as plannings online, criei meu próprio scrum poker 😄 

[https://www.scrum-poker.co/](https://www.scrum-poker.co/)

Fiz junto com um amigo, ambos estamos usando e nossas empresas. Foi feito com Nextjs, MongoDB, Socket.io.

Acessa lá e divirta-se rsrsrsrs

**Comment by sarahdialmeida** [comment]
Que otima ideia, Alan. Vou propor de experimentar lá no meu time.
